pkgs/development/libraries/dbus/default.nix

```diff
@@ -32,14 +35,24 @@ stdenv.mkDerivation rec {
     # Also applied upstream in https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/189,
     # expected in version 1.14
     ./docs-reproducible-ids.patch
-  ] ++ (lib.optional stdenv.isSunOS ./implement-getgrouplist.patch);
+  ] ++ (lib.optional stdenv.isSunOS ./implement-getgrouplist.patch)
+    # AC_PATH_XTRA doesn't seem to find X11 libs even though libX11 seems
+    # to provide valid pkg-config files. This replace AC_PATH_XTRA with
+    # PKG_CHECK_MODULES.
+    # PR open cf https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/212/diffs?commit_id=23880a181e82ee7f
+    ++ lib.optional stdenv.hostPlatform.isStatic
+      (fetchpatch {
+        url = "https://gitlab.freedesktop.org/dbus/dbus/-/commit/6bfaea0707ba1a7788c4b6d30c18fb094f3a1dd4.patch";
+        sha256 = "1d8ay55n2ksw5faqx3hsdpfni3xl3gq9hnjl65073xcfnx67x8d2";
+      });
 
   postPatch = ''
-    substituteInPlace tools/Makefile.in \
-      --replace 'install-localstatelibDATA:' 'disabled:' \
+    substituteInPlace bus/Makefile.am \
```

Comment:
Now you seem to update bus twice and the changes also differ?

Reply:
The disabling of install-localstatelibDATA in tools/Makefile.in was useless, is it fixed now ?